ui/app/routes/jobs/run/index.js

```diff
@@ -69,7 +69,7 @@ export default class RunRoute extends Route {
 
   resetController(controller, isExiting) {
     if (isExiting) {
-      controller.model.deleteRecord();
+      controller.model?.deleteRecord();
```

Comment:
In the event that a user has a bad URL, we will throw an error on fetching the template before the job model is created. So we need some type safety here to account for this situation.